app/src/main/java/net/gsantner/markor/model/Document.java

```diff
@@ -192,6 +194,14 @@ public static Document fromArguments(Context context, Bundle arguments) {
         return document;
     }
 
+    private void setContent(final String s) {
+        _lastContent = s;
```

Comment:
Just storing the content for comparison instead of hash. Simpler and faster. We can try something fancy like storing the hash if content is very large. But in practice, I don't think this will be a problem.

Reply:
what was the reason why we scraped the previous one where we already used & stored hash? just that it's crc32 instead of md5 now

Reply:
I thought about it and I don't think using a hash is any better than just storing the content. 

Options are:
1. crc32 - fast, low chance of collision
2. sha512 - slower, very low change of collision
3. store full text - fastest, 0 chance of collision. But potential extra memory consumption.

Until recently we stored multiple versions of the full content with no issues, so memory isn't a concern in practice. I don't see any downsides to just storing and comparing the entire content.

Reply:
Generally don't think collision isn't something to think about for Markor's use (i.e. if somebody intentionally crafts a md5/sha1 collision) --> not that important, for any of crc/md5/sha1/shaXXX it will give different value if you just type one different character somewhere.

Though have to admit,  we consume quite excess amount of RAM (especially in highlighters), wouldn't wonder if we have like 5 full copies of text floating around somewhere 😄 . By increasing size of file it gets an issue on low-end cheap devices (running into OOM).

Plus: We can't store a copy of the original text in bundle/activity (1MB limit, where ½-¾ of it is already fixed consumed) - regarding device rotations and stuff. Storing some 64char long hash isn't a problem.

**It's not in scope to support it now & in this PR. You can ignore it, but just for your information how my view is on it**

Reply:
> Plus: We can't store a copy of the original text in bundle/activity

Ah. I forgot about this. I will revert to using the hash.